portfolio/src/components/Cover.tsx

```diff
@@ -1,36 +0,0 @@
-import React, { Component } from 'react';
-import profile from "../images/profile.png";
-import Container from 'react-bootstrap/Container';
-
-type Props = {
-
-}
-export default class Cover extends Component<Props>{
-    constructor(props: Props) {
-        super(props);
-    }
```

Comment:
you can get rid of the type here and the constructor, i'm pretty sure. You don't provide props anyways.